jac-cloud/jac_cloud/tests/simple_graph_mini.jac

```diff
@@ -0,0 +1,347 @@
+"""Example of simple walker walking nodes."""
```

Comment:
Is it necessary to have different jac file and openAPI specs for testing mini vs normal jac-cloud? They should be the same right?

Reply:
Mini version only includes walker API generations. Most of the features from jac-cloud will not work